lib/index.js

```diff
@@ -157,16 +158,29 @@ module.exports = function (options, handler) {
   var moesifMiddleware = function (event, context, callback) {
     moesifConfigManager.tryGetConfig();
 
+    const responseTransactionId = event.headers.transactionID || uuid4();
+
```

Comment:
please add bit more safety. what if event.headers is null?

Reply:
@xinghengwang Done.
Just wondering, we check for existing txn ID with `event.headers.transactionID` like above. Should we do it like `event.headers['X-Moesif-Transaction-Id']` instead?

Reply:
yes. we should use event.headers['X-Moesif-Transaction-Id']